src/platform/FreeRTOS/GenericThreadStackManagerImpl_FreeRTOS.ipp

```diff
@@ -127,6 +146,12 @@ void GenericThreadStackManagerImpl_FreeRTOS<ImplClass>::ThreadTaskMain(void * ar
     // Capture the Thread task handle.
     self->mThreadTask = xTaskGetCurrentTaskHandle();
 
+    // Try starting joiner within 15m.
+    self->mJoinerExpire = xTaskGetTickCount() + pdMS_TO_TICKS(15 * 60 * 1000);
+
+    TimerHandle_t joinerTimer = xTimerCreate("JoinerTimer", pdMS_TO_TICKS(10000), pdTRUE, self, &OnJoinerTimer);
```

Comment:
check for NULL return in case of failure?

Reply:
@andy31415 ?